### ISSUE TITLE
Release 1.101.0

### ISSUE THREAD

**Issue by aaemnnosttv** [google/site-kit-wp#6982]
---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Release is prepared, tested, and deployed to wordpress.org.

## Implementation Brief

* N/A

## QA Brief

* N/A

## Changelog entry

* N/A

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

- On view only dashboard if AdSense is shared through Dashboard sharing then I'm getting console error.
`Google Site Kit API Error method:GET datapoint:settings type:modules identifier:adsense error:"Sorry, you are not allowed to do that."` On latest environment, I'm not getting this console error. 

![image](https://github.com/google/site-kit-wp/assets/94359491/3826a794-52d7-4a31-8482-8e9cbef2b2d9)


cc @wpdarren

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

We seem to have some widgets missing on the view only version of the dashboard. 

- Search traffic over the last 28 days
- Top search queries for your site

Neither of these are appearing for me. 

This is a live site.  GA4Reporting is not enabled. 

https://github.com/google/site-kit-wp/assets/73545194/f479d304-8e08-4865-9392-86089b42fa18

**Comment by aaemnnosttv** [comment]
@wpdarren I just tested this and those are both showing for me on the shared dashboard. The problem in your case is that Search Console isn't shared with your user's role as can be seen in the view only menu 
<img width="380" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/0c7b661d-24f7-4289-a4d8-bcd16c6d1ebf">

**Comment by wpdarren** [comment]
@aaemnnosttv yes, I just deleted my comment because I realised I was being stupid. 🤣

**Comment by wpdarren** [comment]
### QA Update: ✅ 

Note: I am able to see the console error that Mohit highlighted above. 

I have completed my release testing:

- Went through our QA checklist on a site in gathering, zero data state, and real data. 
- Didn't complete any GA4 Reporting testing as this will be picked up next week in our final E2E testing. 
- Completed additional testing around AdSense with legacy and V2.
- Tested `dashboardSharing` and `ga4ActivationBanner ` 
- Made sure that `GTESupport` was still functioning as expected.

**Comment by aaemnnosttv** [comment]
I just opened https://github.com/google/site-kit-wp/issues/7064 as a quick fix for something that I noticed while testing. It's not a regression but something we should include in the release which is a very small change that won't invalidate the rest of the QA done here.

**Comment by hussain-t** [comment]
## QA Verified :white_check_mark:
- Smoke tested.
- Verified the changes I’ve been involved with.
- Performed some ad-hoc testing.

**Comment by nfmohit** [comment]
## Release Tested ✅

- Release tested 1.101.0 thoroughly.
- Set everything up from scratch on a live (InstaWP) site.
- Didn't spot any unexpected console errors.
- Verified UI in different device breakpoints.
- Verified changes I've been involved with.
- Used our release testing checklist to ensure everything is checked.

**Comment by tofumatt** [comment]
# QA ✅ 

* Spent time testing with GA4 Reporting _disabled_ and _enabled_.
* Tested GA4 Reporting on mobile viewports.
* Usual smoke and setup flow tests.

Spent a bit more time with this one since it's a big release and we haven't done a release in awhile.

Everything looked good for me. 👍🏻

**Comment by kuasha420** [comment]
## QA: :heavy_check_mark: 

* Went through the release QA testing, set everything up from scratch on a fresh site.
* Tested with and without `gteSupport` and `dashboardSharing` feature flag.
* Tested the changes I was involved with. 

LGTM. :+1:

**Comment by mohitwp** [comment]
@aaemnnosttv Just flagging here the issue which I reported [above](https://github.com/google/site-kit-wp/issues/6982#issuecomment-1554580764). Wondering if it require fix or not ?

**Comment by aaemnnosttv** [comment]
The issue regarding the console error on the view only dashboard is due to the AdSense status migration checks added in https://github.com/google/site-kit-wp/issues/5628 which haven't been made conditionally on the view-only dashboard. We'll want to address that in a follow-up but it need not block the release since the error does not break anything, and a non-admin shouldn't see that migration notice anyways since they wouldn't be able to action it for the same reason.

I'll open an issue to address this, thanks again for flagging 👍
